File: test/Data.Analyzers.Vsix.Test/CSharp/InvalidRegisterLocalColumn.cs
namespace DevZest.Data.Analyzers.Vsix.Test.CSharp
{
    public class InvalidRegisterLocalColumn : Model
    {
        static InvalidRegisterLocalColumn()
        {
            RegisterColumn((InvalidRegisterLocalColumn _) => _.Column1);
        }

        public LocalColumn<string> Column1 { get; private set; }
    }
}

File: test/Data.Analyzers.Vsix.Test/VisualBasic/InvalidRegisterLocalColumn.vb
Public Class InvalidRegisterLocalColumn
    Inherits Model

    Shared Sub New()
        RegisterColumn(Function(x As InvalidRegisterLocalColumn) x.Column1)
    End Sub

    Private m_Column1 As LocalColumn(Of String)
    Public Property Column1 As LocalColumn(Of String)
        Get
            Return m_Column1
        End Get
        Private Set
            m_Column1 = Value
        End Set
    End Property
End Class
